packages/mock-server/src/data/database.ts

```diff
@@ -7,6 +7,12 @@ import {
   NameData,
   removeElement,
 } from './data';
+import {
+  isAlmostExpired,
+  isExpired,
+  isThisWeek,
+  isToday,
+} from '@openmsupply-client/common/src/utils/dateFunctions';
```

Comment:
yeah, not ideal. Importing directly from `common` makes it choke - getting ENVIRONMENT without using node.. I think? can't remember.